wellcomeml/ml/bilstm.py

```diff
@@ -163,7 +163,7 @@ def fit(self, X, Y, embedding_matrix=None, *_):
         steps_per_epoch = math.ceil(X_train.shape[0] / self.batch_size)
         validation_steps = math.ceil(X_val.shape[0] / self.batch_size)
 
-        if tf.config.list_physical_devices('GPU'):
+        if len(tf.config.list_physical_devices('GPU')) > 1:
```

Comment:
will this error if `tf.config.list_physical_devices('GPU')` doesnt exist, or are you confident this will never happen?

Reply:
i tried on my CPU and in that case it runs an empty list